src/components/CreateTemplate/index.tsx

```diff
@@ -0,0 +1,210 @@
+      setInputError((inputError) => ({
+        ...inputError,
+        courtNameNullErr: !courtName,
+      }));
+      return;
+    }
+    const packedTemplate = generateNewTemplate(
+      userId,
+      courtName,
+      description,
+      selectedCourtTileData,
+      selectedCourt
+    );
+    await addTemplate(packedTemplate)
+      .unwrap()
+      .then((_res) => {
+        handleSuccessNoticeOpen();
+      })
+      .catch((err) => {
+        alert("Whoops! unsuccessful publish your template, please try again!");
+      });
+    closeWindow();
+  };
+
+  return (
+    <>
+      <Modal isOpen={isOpen} onClose={closeWindow} size="2xl" isCentered>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader role="modalTitle">Template sharing</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody padding="0.5rem 1.5rem">
+            <FormLabel>Court Preview:</FormLabel>
+            <Flex
+              fontSize="2rem"
+              fontWeight="500"
+              width="20rem"
+              height="11rem"
+              backgroundColor="orange"
+              justifyContent="center"
+              alignItems="center"
+              margin="1.8rem auto"
+            >
+              Court Image
+            </Flex>
+            <Flex gap="1.5rem" justifyContent="center" flexWrap="wrap">
+              <Badge margin="1rem" colorScheme="green">
+                {courtType}
+              </Badge>
+              <Badge margin="1rem">{selectedCourt.courtName}</Badge>
+            </Flex>
+            <Flex>
+              <FormControl
+                width="50%"
+                marginTop="1rem"
+                isRequired
+                isInvalid={inputError.courtNameFullErr}
+              >
+                <FormLabel marginBottom="1rem">Template Court Name:</FormLabel>
+                <Input
+                  role="courtNameInput"
+                  placeholder="Court name"
+                  width="15rem"
+                  onChange={checkNameLength}
+                  autoFocus
+                  ref={courtNameRef}
+                />
+              </FormControl>
+              <Box width="50%">
+                <Text margin="1rem 0px" fontSize="medium" fontWeight="500">
+                  Publisher:
+                </Text>
+                <Flex alignItems="center">
+                  <Icon as={FaUserCircle} fontSize="2.5rem" marginRight="1.8rem" />
+                  <Text
+                    fontSize="large"
+                    fontWeight="500"
+                    whiteSpace="nowrap"
+                    overflow="hidden"
+                    textOverflow="ellipsis"
+                  >
+                    {`${firstName} ${lastName}`}
+                  </Text>
+                </Flex>
+              </Box>
+            </Flex>
+            <ErrorMsg userInputError={inputError} inputErrorMsg={INPUT_ERROR_MSG} />
```

Comment:
> The statement should be wrapped FormControl and set width the same as the input, otherwise if the error message is too long, it will be displayed in the publisher's area
> 
> ![image](https://user-images.githubusercontent.com/76163407/196825693-6a1370ce-58f4-4101-803b-6fec827e03c1.png)

Fixed thanks

Reply:
> The statement should be wrapped FormControl and set width the same as the input, otherwise if the error message is too long, it will be displayed in the publisher's area
> 
> ![image](https://user-images.githubusercontent.com/76163407/196825693-6a1370ce-58f4-4101-803b-6fec827e03c1.png)

Fixed thanks